keylime/cmd/convert_config.py

```diff
@@ -241,12 +367,15 @@ def process_mapping(old_config, templates, mapping_file, debug=False):
             print(f"Failed while running adjustment from {adjust_script}: {e}")
 
     if debug:
-        print(json.dumps(new, indent=4))
+        out = {}
+        for s in new.sections():
+            out[s] = dict(new.items(s))
```

Comment:
why is this needed? is not the configparser behaving like a dict of dicts?

Reply:
`TypeError: Object of type RawConfigParser is not JSON serializable`